SOLUTIONS/yP_ASCIIProduct.py

```diff
@@ -1,6 +1,6 @@
-var= raw_input()
+var= input()
 lvar= list(var)
 x=1
 for a in range(0, len(lvar)):
   x= x*ord(lvar[a])
- print (x)
+ print(x)
```

Comment:
There is an extra space before __print()__  Python is _very_ picky about indentation,

Reply:
aahhh, okay

Reply:
```python
 print(x)  # change this to -->
print(x)
```

Reply:
i did